services/common/src/constants/strings.tsx

```diff
@@ -173,6 +173,7 @@ export const NOTICE_OF_DEPARTURE_STATUS_VALUES = {
   determined_non_substantial: "determined_non_substantial",
   determined_substantial: "determined_substantial",
   withdrawn: "withdrawn",
+  self_authorized: "self_authorized",
```

Comment:
Noticed this was missing and causing an error in mds\services\core-web\src\components\modalContent\NoticeOfDepartureModal.tsx line 203. I just added this here not sure if anyone has anymore context or if this is the correct string value to set for this?